sharding/nodesCoordinator/indexHashedNodesCoordinatorLite_test.go

```diff
@@ -0,0 +1,89 @@
+package nodesCoordinator
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfo(t *testing.T) {
+	t.Parallel()
+
+	arguments := createArguments()
+
+	ihgs, err := NewIndexHashedNodesCoordinator(arguments)
+	require.Nil(t, err)
+	epoch := uint32(1)
+
+	randomness := []byte("rand seed")
+
+	ihgs.nodesConfig[epoch] = ihgs.nodesConfig[0]
+
+	body := createBlockBodyFromNodesCoordinator(ihgs, epoch)
+	validatorsInfo, err := createValidatorInfoFromBody(body, arguments.Marshalizer, 10)
+	require.Nil(t, err)
+
+	err = ihgs.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo)
+
+	validators, err := ihgs.GetAllEligibleValidatorsPublicKeys(epoch)
+	require.Nil(t, err)
+	require.NotNil(t, validators)
```

Comment:
I don't feel like this test truly covers and tests the functionality of the function. 
You don't check anywhere the actual stored new config. Even though you check for error to be nil, how can you be sure that the stored data is the one that you expect?